tools/slack/user-management.md

```diff
@@ -50,3 +50,59 @@ Here are the resources used by Slack Admins to track and manage Slack accounts:
   - [New public user requests (external)](https://docs.google.com/forms/d/e/1FAIpQLSfYQ-D82rIGwbCmwF3kAQERqczi5syVGq6GtmQNR6fhxRAA2Q/viewform?usp=sf_link)
   - [Integration requests](https://docs.google.com/a/gsa.gov/forms/d/1sH-eLcDMDSBE9xvUnbE39N0PFOcfg6Mf3mnWU8xzBz8/edit#responses)
 - [Delegation of functionality](https://docs.google.com/a/gsa.gov/document/d/1gDuScce7R6q6NqQPPS3cFe3dZFYO_ZEp60dmuzVDYwg/edit?usp=sharing)
+- Requests for approved emoji are fulfilled from [#approved-emoji](https://gsa-tts.slack.com/archives/C024G4VLWGM)
+
+
+## Custom emoji
+
+All custom emoji added to Slack must be
+[reviewed](https://github.com/18f/tts-tech-portfolio/issues/1077) by TTS staff.
+Users can request custom emoji following the process described [here]({% link tools/slack/guidelines.md %}#custom-emoji).
+The overall process looks like this.
+
+1. Users request new emoji from [#emoji-showcase](https://gsa-tts.slack.com/archives/C0X2T36AY)
+1. TTS reviews the request in [#admins-emoji](https://gsa-tts.slack.com/archives/C024EBDS1NC)
+1. Approved emoji are routed to [#approved-emoji](https://gsa-tts.slack.com/archives/C024G4VLWGM) for Slack admins
+1. Slack admins [create the approved emoji](https://gsa-tts.slack.com/customize/emoji) in the Slack workspace
+1. Users are notified the new emoji is ready for use
+
+Ideas for automating the creation of approved emoji are discussed in
+[18f/tts-tech-portfolio#1077](https://github.com/18f/tts-tech-portfolio/issues/1077).
+
+
+### Approval criteria
+
+TTS staff in [#admins-emoji](https://gsa-tts.slack.com/archives/C024EBDS1NC) are
+responsible for reviewing new emoji. Emoji are covered by the GSA IT Rules of
+Behavior and the [TTS Code of Conduct]({% link _pages/policies/conduct-policies/code-of-conduct.md %}).
+TTS staff should review emoji requests with these rules in mind.
```

Comment:
```suggestion
TTS staff should review emoji requests with these rules in mind. Submitters should be thoughtful and mindful about emoji submissions, and try to ensure they're in line with our values of diversity and inclusion.
```

Reply:
Trying to put some onus on the submitter, and also make them think about things that may be pretty questionable to demographic and cultural groups other than their own (which is where I think we've generally crossed a line before). It might be good to have someone from the diversity guild give this a once-over to see if they can provide better guidance than I can.